src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operationhandler.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitShopService {
+
+    private Storage storage;
+    private OperationStrategy operationStrategy;
+
+    public FruitShopService(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+        this.storage = new Storage();
+    }
+
+    public void processTransactions(List<FruitTransaction> transactions) {
+        for (FruitTransaction transaction : transactions) {
+            OperationHandler operationHandler = operationStrategy
+                    .getHandler(transaction.getOperation());
+            operationHandler.handleOperation(transaction, storage);
+        }
```

Comment:
same as above, you don't need object of storage class, you call static element.
recheck whole project for this issue (handlers, main class, probably somewhere else)
```suggestion
    private OperationStrategy operationStrategy;

    public FruitShopService(OperationStrategy operationStrategy) {
        this.operationStrategy = operationStrategy;
    }

    public void processTransactions(List<FruitTransaction> transactions) {
        for (FruitTransaction transaction : transactions) {
            OperationHandler operationHandler = operationStrategy
                    .getHandler(transaction.getOperation());
            operationHandler.handleOperation(transaction);
        }
```